hack/gosec.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+export ARTIFACTS=$KUBEVIRT_DIR/${ARTIFACTS:-_out/artifacts}
+if ${GENERATE:-"false"} == "true"; then
+
+    mkdir -p $ARTIFACTS
+
+    echo "Run go sec in pkg"
+    cd $KUBEVIRT_DIR/pkg
+
+    # -confidence=high -severity=high <- for filtering
+    gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -no-fail -exclude-dir=testutils -fmt=junit-xml ./...
+
+    cp ${ARTIFACTS}/junit-gosec.xml ${KUBEVIRT_DIR}/tools/gosec
+else
+    cd ${KUBEVIRT_DIR}/tools/gosec
+    git --no-pager diff "junit-gosec.xml"
```

Comment:
So you are showing the differences here. What I would expect to happen was a check whether a new issue has been discovered, but I don't see that here? Am I missing something?

Reply:
We can store a number of cases for each testsuite and then find out if we added a new one. But it's not perfect since one can fix a testcase but introduce another one... We can't really identify uniquely each testcase(IMO) so I went with the simplest solution... WDYT?

Reply:
I'd like to keep it as simple as possible. So I'd rather not store the junit file and check whether there are new issues from there. I'd rather mark every issue that gosec currently encounters with something like this: 

```
// nosec FIXME <ISSUE TYPE> <SEVERITY>
```

Then we can create issues to fix them one after the other.

Furthermore `gosec` should just fail on every new issue that gets encountered.

Reply:
Currently, there are 300+ issues,  I can probably craft a script which will mark every issue with `nosec`, do you think it's worth the effort? Then the `gosec` can be used to fail for new issues.

Reply:
OK, maybe you are right in that doesn't make sense to mark all the issues with nosec. But still, I'm against storing some reporting output and looking at whether something changed in it. Maybe it's best to just let it fail, let prow collect the junit and work with the deck display as a start? I mean, the check does not yet report, so users will not get annoyed. But we can work with the results already.